src/js/profile/mobile/widget/Calendar.js

```diff
@@ -0,0 +1,835 @@
+							for (var idx in this.globalObj) {
+								if (this.globalObj[idx].input != null && this.globalObj[idx].input.value.length > 0) {
+									var convertDate = this._strToDate(this.globalObj[idx].input.value, this._dateFormat || this.langData[this.lang].dateformat || "%Y-%m-%d");
+
+									if (convertDate instanceof Date) {
+										obj[idx] = convertDate;
+									}
+								}
+							}
+						}
+						this._dateFormat = dateFormat;
+						var regexpFunc = String(this._dateFormat).replace(/%[a-zA-Z]+/g, function (str) {
+							var inputCase = str.replace(/%/, "");
+
+							switch (inputCase) {
+								case "n":
+								case "h":
+								case "j":
+								case "g":
+								case "G":
+									return "\\d{1,2}";
+								case "m":
+								case "d":
+								case "H":
+								case "i":
+								case "s":
+								case "y":
+									return "\\d{2}";
+								case "Y":
+									return "\\d{4}";
+								case "M":
+									return "(" + self.langData[self.lang].monthesSNames.join("|").toLowerCase() + "){1,}";
+								case "F":
+									return "(" + self.langData[self.lang].monthesFNames.join("|").toLowerCase() + "){1,}";
+								case "D":
+									return "[a-z]{2}";
+								case "a":
+								case "A":
+									return "AM|PM";
+								case "u":
+									return "\\d{1,6}";
+								case "P":
+									return "[\\+\\-]\\d{1,2}\\:\\d{1,2}"
+							}
+							return str;
+						});
+
+						this._dateFormatRE = new RegExp(regexpFunc, "i");
+						for (var data in obj) {
+							this.globalObj[data].input.value = this._dateToStr(obj[data]);
+						}
+						obj = null;
+					};
+					this.setDateFormat(this.langData[this.lang].dateformat || "%Y-%m-%d");
+					this.getWeekNumber = function (dayOfMon) {
+						// The first start of the calendar is counted from Monday.
+						if (typeof (dayOfMon) == "string") {
+							dayOfMon = this._strToDate(dayOfMon);
+						}
+						if (!(dayOfMon instanceof Date)) {
+							return "Invalid Date";
+						}
+						var monCnt = dayOfMon.getDay();
+
+						if (monCnt === 0) {
+							monCnt = 7
+						}
+						var cloneMonday = new Date(dayOfMon.valueOf());
+
+						cloneMonday.setDate(dayOfMon.getDate() + (4 - monCnt));
+						var currentYear = cloneMonday.getFullYear();
+						var yearOfDayCnt = Math.round((cloneMonday.getTime() - new Date(currentYear, 0, 1).getTime()) / 86400000);
+
+						return 1 + Math.floor(yearOfDayCnt / 7);
+					};
+					this.show = function (flag) {
+						this._show(flag);
+					};
+					this.hide = function () {
+						if (this._isVisible()) {
+							this._hide();
+						}
+					};
+					this.isVisible = function () {
+						return this._isVisible();
+					};
+					this.pos = "bottom";
+					this._show = function (flag) {
+						if (flag === true && this._isVisible()) {
+							this._hide();
+							return;
+						}
+						this.base.style.visibility = "hidden";
+						this.base.style.display = "";
+						if (this._px != null && this._py != null) {
```

Comment:
How about to move the style code to less file?

Reply:
Styles that change on condition cannot be applied to less.